### PR TITLE
RDISCROWD-4878 ask browse bulk edit state not updating

### PR DESCRIPTION
**Pull request by XiChenn**
*Issue number of the reported bug or feature request: RDISCROWD-4878*

**Describe your changes**
- When clicking bulk update dropdown button or bulk delete button, reset local selectedTask variable as well as the state.

**Testing performed**
Tested locally.

**Additional context**
related pr: https://github.com/bloomberg/pybossa/pull/777

### CODE REVIEW

File: static/js/tasks_browse.js [hunk 1]
```diff
@@ -291,8 +291,14 @@ $(document).ready(function() {
         showAssignWorkerModal()
     });
 
-    $('#delete-tasks-modal .cancel-delete').on('click', function() {
+    // reset the local variable as well as the state
+    function retSelectedTask() {
         selectedTask = undefined;
+        taskBrowse.setSelectedTask(undefined);
```
Comment (kbecker42) [inline static/js/tasks_browse.js:297]: Also consider: `taskBrowse.setSelectedTask(selectedTask);`

File: static/js/tasks_browse.js [hunk 2]
```diff
@@ -291,8 +291,14 @@ $(document).ready(function() {
         showAssignWorkerModal()
     });
 
-    $('#delete-tasks-modal .cancel-delete').on('click', function() {
+    // reset the local variable as well as the state
+    function retSelectedTask() {
```
Comment (kbecker42) [inline static/js/tasks_browse.js:295]: Would a more appropriate name for this method be `resetSelectedTask()`?

**Review by kbecker42 — COMMENTED** [review]
Thank you.

File: static/js/tasks_browse.js (hunk 2 above)
Comment (XiChenn) [reply static/js/tasks_browse.js:295]: It is. It was a typo :(

File: static/js/tasks_browse.js (hunk 1 above)
Comment (XiChenn) [reply static/js/tasks_browse.js:297]: Will do, thank you.

**Review by kbecker42 — APPROVED** [review]

**Pull request merged**